code/app/CMakeLists.txt~

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.6)
```

Comment:
This file shall not be part of the repository.
Fine tune your `.gitignore` to skim them out.